src/minisignxml/sign.py

```diff
@@ -26,6 +26,8 @@ def sign(
         element_id = element.attrib["ID"]
     except KeyError:
         raise NoIDAttribute(element)
+    if not isinstance(element_id, str):
+        raise TypeError()
```

Comment:
[nitpick] Does TypeError() take a string? My gut instinct is to always feed some kind of error message into it that gives a little more context on the specific type error.

Reply:
it can take a string (or actually anything you want) but this is once again more so to make the type checker happy (since `element.attrib` is a `dict[str, bytes | str]`), byte attributes are basically not supported or expected. the stacktrace you get should point you at what went wrong though even without a message

Reply:
Understood. I tend to prefer a message for readability, but I understand why it's not necessary in this case.